src/config/options.schema.json

```diff
@@ -1826,6 +1826,13 @@
           "description": "Output ARG as dot file.",
           "type": "boolean",
           "default": false
+        },
+        "argdotlabel": {
```

Comment:
Since we now have three ARG related options, should we maybe group them under `exp.arg.*`?